tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -314,6 +314,11 @@ class WskBasicUsageTests
             }, waitBeforeRetry = Some(1.second), N = 5)
     }
 
+    it should "include CLI user agent headers with outbound requests" in {
+        val stdout = wsk.cli(Seq("action", "list", "-v", "--auth", wskprops.authKey) ++ wskprops.overrides).stdout
+        stdout should include regex ("""\bUser-Agent\b": \[(\r\n|\r|\n)\s+"OpenWhisk\-CLI/201.*""".r)
```

Comment:
Little simpler to just check for whitespace instead of the carriage returns and new lines. Also seems like the test will fail in 2020.`"""\bUser-Agent\b": \[\s+"OpenWhisk\-CLI/\d+.*""".r`

Reply:
yes, you're right about 2020, I chose that to follow suit with the rest of the CLI version tests in this file. I will test a cleaner regex out